main.go

```diff
@@ -96,41 +96,42 @@ func setNpmVersion(ver string) (string, error) {
 	return out, nil
 }
 
-func readFromPackageJSON(workdir string) string {
-	path := filepath.Join(os.Getenv("ORIG_BITRISE_SOURCE_DIR"), "package.json")
+func readFromPackageJSON(workdir string) (string, error) {
+	path := filepath.Join(workdir, "package.json")
 	ver, err := getNpmVersionFromPackageJSON(path)
```

Comment:
as you do not modify the putputs of `getNpmVersionFromPackageJSON `, just return them `return getNpmVersionFromPackageJSON`